nemo/collections/llm/evaluation/eval_utils.py

```diff
@@ -0,0 +1,285 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can we rename this file to `base.py` to keep it consistent with other folders?

Reply:
Done in the latest commit.